source/guide_uu-notify.rst

```diff
@@ -0,0 +1,275 @@
+
+License
+=======
+
+All relevant legal information is mentioned in uuNotify's LICENSE_.
+
+Prerequisites
+=============
+
+Node.js version
+---------------
+
+We're using :manual:`Node.js <lang-nodejs>` version 14.
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ uberspace tools version use node 14
+  Selected Node.js version 14
+  The new configuration is adapted immediately. Minor updates will be applied automatically.
+  [isabell@stardust ~]$
+
+
+Gotify Server
+-------------
+
+You have installed and configured your gotify server, :lab:`as described in the respective guide <guide_gotify>`.
+
+Limitations
+===========
+
+Since uuNotify uses the GitHub API, only software available on GitHub is checked for updates. It is planned to support other release feeds in a future version of uuNotify (see `uuNotify GitHub issues <https://github.com/franok/uu-notify/issues/2>`_).
+
+Installation
+============
+
+Clone the `GitHub repository <https://github.com/franok/uu-notify>`_, checkout the latest_ version and run the setup script.
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ UUN_VERSION=v0.0.0
+  [isabell@stardust ~]$ git clone https://github.com/franok/uu-notify
+  [isabell@stardust ~]$ cd uu-notify/
+  [isabell@stardust uu-notify]$ git checkout $UUN_VERSION
+  [isabell@stardust uu-notify]$ git branch
+  * (HEAD detached at <latest version>)
+    main
+  [isabell@stardust uu-notify]$ ./setup.sh
+  --- uuNotify setup ---
+  Existing files in config/ will not be overwritten.
+  
+  Running 'npm clean-install' ...
+  [...]
+  Done.
+  
+  Copy configuration files...
+  Creating config.json from config.json.template ...
+  Done.
+  Creating software-deps.mjs from software-deps.mjs.example ...
+  Done.
+  
+  --- COMPLETED uuNotify setup ---
+  [isabell@stardust uu-notify]$
+
+Configuration
+=============
+
+Gotify application
+------------------
+
+Create a gotify application.
+Log in to your Gotify server's WebUI, click the ``Apps``-tab in the menu bar and create an application. An app token is generated automatically. You'll need it in the next step.
+
+
+Configure uuNotify
+------------------
+
+In the uuNotify directory, navigate into the ``config/`` folder:
+
+.. code-block:: console
+
+  [isabell@stardust uu-notify]$ cd config/
+  [isabell@stardust config]$ 
+
+Edit the file ``config.json``.
+
+Add your gotify server url and the app token from the previous step. Example:
+
+.. code-block:: json
+
+  {
+    "gotify": {
+        "url": "https://isabell.uber.space/gotify",
+        "token": "AbccRsTUvwXX5yQ"
+    },
+    [...]
+  }
+
+.. note:: The gotify server URL must end **without** trailing forward slash ``/``
+  
+  | Good example: ``https://isabell.uber.space/gotify``
+  | Bad example: ``https://isabell.uber.space/gotify/``
+
+
+Now add your `personal GitHub access token <https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token>`_. Example:
```

Comment:
A hint that the token only needs access to the notifications would be appropriate here.